app/src/test/java/dev/arkbuilders/navigator/data/utils/PathExtKtTest.kt

```diff
@@ -0,0 +1,88 @@
+package dev.arkbuilders.navigator.data.utils
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.name
+import kotlin.io.path.notExists
+
+@RunWith(MockitoJUnitRunner::class)
+class PathExtKtTest {
+
+    private lateinit var testee: Path
+
+    @BeforeEach
+    fun setUp() {
+        testee = Paths.get("notAndroid")
+    }
+
+    @Test
+    fun givenPath_whenFindNotExistCopyNameFalse_thenReturnNewPath() {
+        val sourcePath = mockk<Path>()
+        val destPath = mockk<Path>()
+
+        every { destPath.fileName } returns Paths.get("notAndroid")
+        every { sourcePath.resolve(Paths.get("notAndroid")) } returns Paths.get("notAndroid")
+        every { sourcePath.resolve(Paths.get("notAndroid")).notExists() } returns false
+
+        every { sourcePath.resolve("notAndroid_1.") } returns Paths.get("notAndroid_1.")
+
+        val result = sourcePath.findNotExistCopyName(destPath)
+
+        assertEquals("notAndroid_1.", result.name)
```

Comment:
Maybe just `"A"` and `"B"` would be easier to read instead of `notAndroid` and `not_Android_1.`..